extension/test/test-logger.js

```diff
@@ -27,7 +33,12 @@ var verifyAsyncOutput = function (test, logger, testFunc) {
     }
     finally {
       istream.close();
-      message = JSON.parse(message);
+      try {
+        message = JSON.parse(message);
```

Comment:
As you can see currently `try ... catch ... Cu.reportError` is used to get that bad strings to console. Are there any test framework function for the same?